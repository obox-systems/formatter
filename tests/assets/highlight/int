123
hello
0b0011u32
-2.5e-3
1_000_000u32
