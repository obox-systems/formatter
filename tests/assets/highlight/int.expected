[40m1[0m[40m2[0m[40m3[0m[101m
[0m[42mhello[0m[101m
[0m[40m0[0m[42mb0011u32[0m[101m
[0m[42m-[0m[40m2[0m[40m.[0m[40m5[0m[42me[0m[42m-[0m[40m3[0m[101m
[0m[40m1[0m[42m_000_000u32[0m[101m
[0m